src/app/app.js

```diff
@@ -1,5 +1,6 @@
 angular.module('prx', ['ngAnimate',
   'ngSanitize',
+  'prx.ui',
```

Comment:
We're actually making use of the stuff in the ui-module in our application now.

I've talked a little bit with farski about the distinction between `app/ui`, `app/[stories]`, and `common/[module]`, and I think it makes sense to us, but please let me know if it's not obvious. Basically, `app/ui` ends up being where we keep things that sort of fit in the realm of jquery-ui (like nav, tabs, etc) that may be used across multiple modules on the site, but that are also very application specific and not likely candidates for extraction.